README.md

```diff
@@ -143,6 +178,134 @@ has_audit_log enqueue_with: { priority: 30 }
 # Or for custom journaling:
 journal_attributes :email, enqueue_with: { priority: 20, queue: 'journaled' }
 ```
+##### Database-Backed Event Processing (Optional)
```

Comment:
The DJ implementation is also enforced to be DB-backed, so maybe this could be re-framed as a dedicated Outbox feature.